provisioner/src/lib.rs

```diff
@@ -581,6 +582,16 @@ impl Provisioner for MyProvisioner {
         Ok(Response::new(reply))
     }
 
+    #[tracing::instrument(skip(self))]
+    async fn provision_qdrant(
+        &self,
+        _request: Request<QdrantRequest>,
+    ) -> Result<Response<QdrantResponse>, Status> {
+        Err(Status::invalid_argument(
+            "Provisioning Qdrant on Shuttle is not supported",
+        ))
+    }
```

Comment:
The fact that this should only be used locally and not in the provisioner makes me think that this should rather be an extension trait, that's only used where it's needed.

Reply:
Yep, the other changes suggested might prompt more differences between local and deployed provisioner.

Reply:
Extension trait was not possible due to resource builder taking `&mut dyn Factory`, no big deal